app/celery/scheduled_tasks.py

```diff
@@ -189,6 +190,25 @@ def replay_created_notifications():
         for n in notifications_to_resend:
             send_notification_to_queue(notification=n, research_mode=n.service.research_mode)
 
+    # if the letter has not be send after 4 hours + 15 minutes, then create a zendesk ticket
+    letters = letters_missing_from_sending_bucket(resend_created_notifications_older_than)
+
+    if len(letters) > 0:
+        msg = "{} letters were created four hours and 15 minutes ago, " \
+              "but do not have an updated_at timestamp or billable units. " \
+              "It is likely you need to run the " \
```

Comment:
I'm liking that this ticket has how to action included on it. Would be nice to make the suggested action a little bit more precise if possible. I'm guessing something along the lines of:

- First check if the letter does exist in the sending bucket (optionally saying how to do this if not obvious or if not clear what the name of the bucket is)
- What to do in either case when the letter does or does not exist in the bucket.

I'm fine with it just being a zendesk ticket at the moment, as long as we keep an eye on if we can swap to automate it later to just rerun the task if we are happy that is what should happen.